queries/modules/io/28/modules/http-client-wrapper.jq

```diff
@@ -0,0 +1,90 @@
+module namespace http = "http://28.io/modules/http-client-wrapper";
+
+import module namespace base64 = "http://zorba.io/modules/base64";
+
+import module namespace http-impl = "http://zorba.io/modules/http-client";
+
+declare namespace ver = "http://zorba.io/options/versioning";
+
+declare variable $http:default-boundary := "multipart/mixed; boundary=DEFAULT_BOUNDARY_14201231297125830186";
```

Comment:
@gislenius This is a hack because qbe response can contain maybe `Content-Type` headers in the same response. It looks like a bug in the ML REST API.